contracts/VeJoeToken.sol

```diff
@@ -36,6 +38,8 @@ contract VeJoeToken is VeERC20("VeJoeToken", "veJOE"), Ownable {
     function setBoostedMasterChefJoe(address _boostedMasterChef) external onlyOwner {
         // We allow 0 address here if we want to disable the callback operations
         boostedMasterChef = IBoostedMasterChefJoe(_boostedMasterChef);
+
+        emit UpdateBoostedMasterChefJoe(msg.sender, _boostedMasterChef);
```

Comment:
Use `_msgSender()`

Reply:
This contract doesn't inherit from `Context` so unless we change that we can't use `_msgSender` here.

Reply:
It does inherit from `Context` as `Ownable` inherits it.